.github/files/generate-ci-matrix.php

```diff
@@ -80,6 +80,15 @@
 	);
 }
 
+// todo: move to the main matrix once WP latest is compatible with PHP 8.5.
+$matrix[] = array(
+	'name'    => 'PHP tests: PHP 8.5 WP trunk',
+	'script'  => 'test-php',
+	'php'     => '8.5',
+	'wp'      => 'trunk',
+	'timeout' => 20,
```

Comment:
Since we're not running a WP latest check with PHP 8.5, let's force the package tests to run with this block.
```suggestion
	'timeout' => 20,
	'force-package-tests' => true,
```

Reply:
Good call. Looks like there are some additional items that need fixing.